.github/workflows/pr.yml

```diff
@@ -20,6 +20,8 @@ jobs:
         run: |
           make minikube-linux-amd64
           make e2e-linux-amd64
+          make kic-base-image
```

Comment:
we should only do this only if the Dockerfile been changed. if not it can cause all sort of issues. currently our base ubuntu image is not pinned and each time we get a different installed versions. not good idea for a reliable idea of testing what we ship to the users.